envs/centos7/py36/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM centos:7
+
+RUN yum install -y centos-release-scl-rh epel-release
+
+RUN yum install -y git cmake3 ninja-build devtoolset-8-toolchain \
+rh-python36-python rh-python36-python-devel rh-python36-python-pip
+
+RUN alternatives --install /usr/local/bin/cmake cmake /usr/bin/cmake3 20 \
+--slave /usr/local/bin/ctest ctest /usr/bin/ctest3 \
+--slave /usr/local/bin/cpack cpack /usr/bin/cpack3 \
+--slave /usr/local/bin/ccmake ccmake /usr/bin/ccmake3 \
+--family cmake
+
+USER 997:995
```

Comment:
The naked USER directive is a bit odd without a corresponding RUN that adds a user and group with a corresponding uid/gid to the container image.

Reply:
agreed, see above